apps/web/modules/ee/contacts/attributes/actions.ts

```diff
@@ -0,0 +1,176 @@
+"use server";
+
+import { z } from "zod";
+import { ZId } from "@formbricks/types/common";
+import { authenticatedActionClient } from "@/lib/utils/action-client";
+import { checkAuthorizationUpdated } from "@/lib/utils/action-client/action-client-middleware";
+import { AuthenticatedActionClientCtx } from "@/lib/utils/action-client/types/context";
+import {
+  getOrganizationIdFromEnvironmentId,
+  getProjectIdFromEnvironmentId,
+} from "@/lib/utils/helper";
+import { isSafeIdentifier } from "@/lib/utils/safe-identifier";
+import { withAuditLogging } from "@/modules/ee/audit-logs/lib/handler";
+import {
+  createContactAttributeKey,
+  updateContactAttributeKey,
+  deleteContactAttributeKey,
+  getContactAttributeKeyById,
+} from "@/modules/ee/contacts/lib/contact-attribute-keys";
+
+const ZCreateContactAttributeKeyAction = z.object({
+  environmentId: ZId,
+  key: z.string().refine(
+    (val) => isSafeIdentifier(val),
+    {
+      message:
+        "Key must be a safe identifier: only lowercase letters, numbers, and underscores, and must start with a letter",
+    }
+  ),
+  name: z.string().optional(),
+  description: z.string().optional(),
+});
+
+export const createContactAttributeKeyAction = authenticatedActionClient
+  .schema(ZCreateContactAttributeKeyAction)
+  .action(
+    withAuditLogging(
+      "created",
+      "contactAttributeKey",
+      async ({ ctx, parsedInput }: { ctx: AuthenticatedActionClientCtx; parsedInput: Record<string, any> }) => {
```

Comment:
please use the correct type of the parsedInput from the zod schema and remove the `any` type from here 🙏